### ISSUE TITLE
Using form.end helper with more than one buttons causes spacing issues

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#44]
Reported by @james-manley 

```
{%
    set formActions = [
        html.button({
             'name': 'continue',
             'label': 'Go to Payment',
             'class': 'btn--primary'
        }),
        html.button({
            'type': 'link',
            'href': path('paybridge_moto_payment_review', {'cancel': 'true'}),
            'label': 'Cancel Order',
            'class': 'btn--danger'
        })
    ]
%}
{{
    form.end({
        'actions': formActions
    })
}}
```

Actual:
<img width="476" alt="screen shot 2016-02-17 at 11 21 17" src="https://cloud.githubusercontent.com/assets/18653/13108060/b923fbe8-d568-11e5-8c1a-f075d6538aa7.png">

Expected:
<img width="346" alt="screen shot 2016-02-17 at 11 21 26" src="https://cloud.githubusercontent.com/assets/18653/13108049/ac07f6bc-d568-11e5-8eff-0d3b7d247037.png">

**Comment by Stanton** [comment]
``` {%
            set formActions = [
                html.button({
                     'name': 'continue',
                     'label': 'Go to Payment',
                     'class': 'btn--primary'
                }),
                html.button({
                    'type': 'link',
                    'href': path('paybridge_moto_payment_review', {'cancel': 'true'}),
                    'label': 'Cancel Order',
                    'class': 'btn--danger'
                })
            ]
        %}
        {{
            form.end({
                'actions': formActions
            })
        }}```

**Comment by Stanton** [comment]
Fixed by #43
